README.rst

```diff
@@ -88,7 +88,7 @@ Please note that on Linux installing voice you must install the following packag
 * python-dev (e.g. ``python3.8-dev`` for Python 3.8)
 
 
-Quick Example
+Quick Example (with commands.Bot)
 ~~~~~~~~~~~~~
```

Comment:
This line needs to be extended to the same length as the heading.

But I don't think it is necessary to include "(with commands.Bot)"

Reply:
I think I added ooliver's changes.

Reply:
but what is the need to explain to new users what class an example is using

Reply:
Just to help them with differences and compare both classes before they choose a class?